controllers/articles_controller.js

```diff
@@ -20,3 +24,15 @@ exports.getArticleById = (req, res, next) => {
       next(err);
     });
 };
+
+exports.patchArticleVotes = (req, res, next) => {
+  const { article_id } = req.params;
+  const inc_votes = req.body.inc_votes;
+  updatedArticleVotes(article_id, inc_votes)
+    .then((article) => {
+      res.status(200).send(article);
```

Comment:
send this back on a key of article instead -> {article}

Reply:
can you clarify this please? we need to send back the updated article - it is exactly what we have now...